qt/PrefsDialog.h

```diff
@@ -22,13 +22,58 @@ class QString;
 class Prefs;
 class Session;
 
+struct FilterData
```

Comment:
_(nit)_ since the word "Filter" is already so overused in transmission-qt, could you pick a different name for this struct and the following class?